app/src/main/java/com/example/keniphhomemaintenance/maintenance_screen/MaintenanceItemDialog.kt

```diff
@@ -103,7 +105,8 @@ fun NewMaintenanceItemDialog(
 
                     ExtendedFloatingActionButton(
                         onClick = {
-                            maintenanceViewModel.addMaintenanceItem(itemTitle.value, dueDate.value, location.value)
+                            val newMaintenanceItem = MaintenanceItem(PLACEHOLDER_ID, R.drawable.ic_baseline_build_24_light, itemTitle.value, dueDate.value, location.value, dwellingID)
+                            maintenanceViewModel.addMaintenanceItem(newMaintenanceItem)
```

Comment:
wondering if you need to create a val for newMaintenanceItem since it's only used one time